docs/bcfusion/users/new-stake.md

```diff
@@ -41,7 +41,10 @@ To interact with the dApp, first connect your web3 wallet. Currently, `TrustWall
 
 Within the `My Staking` page, you can manage your current delegations.
 
-- Click `Redelegate` to shift your stake to a different validator. Note: A redelegation fee of 0.02% applies to discourage frequent switching between validators.
+- Click `Redelegate` to shift your stake to a different validator. 
+
+Note: A redelegation fee applies to discourage frequent switching between validators. 
```

Comment:
Can we just mention the mainnet? I think most reader dont care about testnet.

Reply:
and we will update the testnet to 0.002% eventually

Reply:
sure, let me update